3 manual continued (#88)
cardonabits/haxo-hw: docs/manual/manual.md

		+ [Install the Newest Binary](#install-newest-binary)
		+ [Howto Change the Sound Set](#howto-change-sound-set)
		+ [Howto Change the notemap.json](#howto-notemap.json)

> 		+ [Howto trigger an action using fingering](#howto-tigger-action)

Reviewer: ```suggestion
		+ [The non-existent Shutdown command](#howto-tigger-shutdown)
```

   Blow air through the mouthpiece or tip of the neck to begin playing.

> 4. **Stop Playing:**

Reviewer: With the new section you are adding right below this, this section should probably be removed.
```suggestion
```

> When you have finished playing and want to power off the haxophone, just remove the USB power. As the SD card is used in read only mode, there is no danger in corrupting the data on the SD card when removing power.
> 
>  <a name="stop-haxophone"></a>
> 
> #### Shutdown the Haxophone
> To turn off the Haxophone you just need to unplug the power. You don't need to worry about corrupting files on the SD card, as the SD card is read only per default.

Reviewer: ```suggestion
 <a name="stop-haxophone"></a>

#### Shutdown the Haxophone
To turn off the Haxophone you just need to unplug the power. You don't need to worry about corrupting files on the SD card, as the SD card is read only per default.
```

 --notemap-file /usr/share/haxo/notemap.json \
 --notemap-file /usr/share/haxo/notemap_c_instrument.json \

> <a name="#howto-tigger-action"></a>
> 
> #### Howto trigger an action using fingering
> 
> TODO Work in Progress....
> 
> Some time ago there existed a [key combination to trigger the shutdown](https://github.com/cardonabits/haxo-rs/commit/9a9a84b224fc8a8fe0d9163508b1aee2d814d735).
> 
> 

Reviewer: I would not introduce a new term (`actions`) given that we already have `commands`.  Also, I would not title a section `How to do THIS` just to say in the body that `THIS` is not possible...  🤔😁)

```suggestion
<a name="#howto-tigger-shutdown"></a>

#### The non-existent Shutdown command

Some time ago there existed a [shutdown command](https://github.com/cardonabits/haxo-rs/commit/9a9a84b224fc8a8fe0d9163508b1aee2d814d735) but it was removed as it was easy to trigger accidentally while playing.
Now that we have more robust method to enter Command Mode that is much harder to trigger accidentally, maybe we should bring back the shutdown command.  If you want it, feel free to open an issue, or even better, a Pull Request.
```

| Document Version | Changes |
| --- | --- |

> | 0.3 | Added howto shutdown  |

Reviewer: Normally this should be a version table showing all versions, not just the last change...
